third_party/accessibility/ax/ax_tree_id_registry.cc

```diff
@@ -2,24 +2,25 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "ui/accessibility/ax_tree_id_registry.h"
+#include "ax_tree_id_registry.h"
 
-#include "base/memory/singleton.h"
-#include "base/strings/string_number_conversions.h"
-#include "ui/accessibility/ax_action_handler_base.h"
+#include "flutter/fml/logging.h"
+
+#include "ax_action_handler_base.h"
 
 namespace ui {
 
 // static
-AXTreeIDRegistry* AXTreeIDRegistry::GetInstance() {
-  return base::Singleton<AXTreeIDRegistry>::get();
+AXTreeIDRegistry& AXTreeIDRegistry::GetInstance() {
+  static AXTreeIDRegistry INSTANCE;
+  return INSTANCE;
 }
 
 void AXTreeIDRegistry::SetFrameIDForAXTreeID(const FrameID& frame_id,
                                              const AXTreeID& ax_tree_id) {
   auto it = frame_to_ax_tree_id_map_.find(frame_id);
   if (it != frame_to_ax_tree_id_map_.end()) {
-    NOTREACHED();
```

Comment:
FML_UNREACHABLE()

Reply:
Ah I didn't realize we have that, thanks for reminding